sdk/keyvault/azure-security-keyvault-secrets/src/test/java/com/azure/security/keyvault/secrets/SecretClientTestBase.java

```diff
@@ -248,6 +248,9 @@ void recoverDeletedSecretRunner(Consumer<KeyVaultSecret> testRunner) {
     @Test
     public abstract void recoverDeletedSecretNotFound(HttpClient httpClient, SecretServiceVersion serviceVersion);
 
+    @Test
+    public abstract void recoverDeletedSecretNotFoundWithPoller(HttpClient httpClient, SecretServiceVersion serviceVersion);
```

Comment:
Same comment as above.
```suggestion
    public abstract void recoverDeletedSecretNotFoundWithPollingDuration(HttpClient httpClient, SecretServiceVersion serviceVersion);
```